### ISSUE TITLE
Bug: running "make lint" reformats the code

### ISSUE THREAD

**Issue by sfermigier** [litestar-org/litestar-fullstack#75]
### Description

Running `make lint` seems to format the code.

After a `make lint`, the following files are modified:

```
	modified:   src/app/cli.py
	modified:   src/app/contrib/aiosql/service.py
	modified:   src/app/contrib/vite/config.py
	modified:   src/app/contrib/vite/loader.py
	modified:   src/app/domain/accounts/controllers/accounts.py
	modified:   src/app/domain/accounts/guards.py
	modified:   src/app/domain/accounts/services.py
	modified:   src/app/domain/plugins.py
	modified:   src/app/domain/tags/controllers.py
	modified:   src/app/domain/teams/guards.py
	deleted:    src/app/domain/web/public/.gitkeep
	modified:   src/app/lib/db/base.py
	modified:   src/app/lib/db/migrations/versions/2023-07-14_initial_models_ddc1b07aa56f.py
	modified:   src/app/lib/dependencies.py
	modified:   src/app/lib/dto.py
	modified:   src/app/lib/repository.py
	modified:   src/app/lib/service/sqlalchemy.py
	modified:   src/app/lib/settings.py
	modified:   src/app/lib/worker/controllers.py
	modified:   src/app/lib/worker/info.py
	modified:   src/app/lib/worker/plugin.py
	modified:   src/app/utils.py
	modified:   tests/conftest.py
	modified:   tests/docker_service.py
	modified:   tests/integration/conftest.py
	modified:   tests/integration/test_access.py
	modified:   tests/integration/test_accounts.py
	modified:   tests/integration/test_teams.py
	modified:   tests/unit/lib/test_crypt.py
	modified:   tests/unit/lib/test_db.py
	modified:   tests/unit/lib/test_dependencies.py
	modified:   tests/unit/test_cli.py
```

I think it's too surprising to have a "make lint" action reformat the code. 

### Steps to reproduce

```bash
Type `make lint`
```

### Litestar Version

The one specified in the project

### Platform

- [ ] Linux
- [X] Mac
- [ ] Windows
- [ ] Other (Please specify in the description above)

**Comment by cofin** [comment]
Should we rename this to `make tidy`?
@JacobCoffee @provinzkraut

**Comment by sfermigier** [comment]
I'd suggest `make format`.

FYI: In my makefiles, I usually have the following standard targets:

```
  develop       Install development dependencies and pre-commit hook (env must be already activated)
  test          Run python tests
  lint          Lint / check typing
  format        Format / beautify code
  clean         Cleanup repository (removes .pyc, caches, etc.)
  tidy          Cleanup harder (e.g. removes .tox, .nox, etc.)
  ...
```

**Comment by JacobCoffee** [comment]
+1 to format

**Comment by provinzkraut** [comment]
> +1 to format

From me as well

**Comment by cofin** [comment]
closed by #120
